app/web/ci-build-protos.sh

```diff
@@ -0,0 +1,26 @@
+# download deps
```

Comment:
so, just some doubts...

this gets the protos right? then it unzips it so they are ready to use, this is the only thing (apart of upgrading the node version) that was needed to enable vercel? (just out of curiosity) is yes, I guess PR looks good

another doubts, this file is called `ci-build-protos`, shouldn't this file being called somewhere? (like I don't see any reference to this file, I guess this is part of the Vercel magic?)

Reply:
in case that both things are a yes, then it'll approved right away

Reply:
It gets the tools to build the protos, then builds them. It doesn't download the protos themselves, they are in `//app/proto`.

`ci-build-protos.sh` is set up in vercel config.

Part of the reason why there was so little to do was that this was already set up before and was just disabled!